core/interop/pinvoke/marshaling/readme.md

```diff
@@ -1,26 +1,35 @@
-.NET Core PInvoke Marshaling Sample
-================
+---
+languages:
+- csharp
+- cpp
+products:
+- dotnet-core
+page_type: sample
+name: "P/Invoke Marshaling Sample"
+urlFragment: "pinvoke-marshal-arguments"
+description: "A .NET application that demonstrates different ways to marshal arguments to native functions when using P/Invokes."
+---
 
-This project demonstrates different ways to marshal arguments to native function when using PInvokes. Documentation can be found here for [PInvokes](https://docs.microsoft.com/dotnet/standard/native-interop/pinvoke) and here for the [Type marshaling](https://docs.microsoft.com/dotnet/standard/native-interop/type-marshaling).
+# .NET Core P/Invoke Marshaling Sample
 
-Prerequisites
-------------
+This project demonstrates different ways to marshal arguments to native functions when using P/Invokes. Documentation can be found here for [P/Invokes](https://docs.microsoft.com/dotnet/standard/native-interop/pinvoke) and here for the [Type marshaling](https://docs.microsoft.com/dotnet/standard/native-interop/type-marshaling).
 
-* .NET Core 3.0 (at least Preview 6) - [https://dot.net](https://github.com/dotnet/core-sdk#installers-and-binaries)
+## Prerequisites
+
+* [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download) (at least Preview 6)
```

Comment:
At this point, should we recommend .NET Core SDK 3.1 (It's the LTS release, and 3.0 is already out of support).

Reply:
That makes sense to me. Updated.